ios/Exponent/Versioned/Modules/Api/GL/ARKit/EXGLARSessionManager.m

```diff
@@ -248,6 +251,42 @@ - (NSDictionary *)rawFeaturePoints
            };
 }
 
+- (NSDictionary *)planes
+{
+  if (!self.arSession) {
+    return nil;
+  }
+  
+  NSArray<ARAnchor *> *anchors = self.arSession.currentFrame.anchors;
+  NSMutableArray *planes = [NSMutableArray array];
+  
+  for (int i = 0; i < anchors.count; i++) {
```

Comment:
Why not `for ... in` ?

Reply:
Forgot about fast enums, yes it would be better that way.